apis/speech/v1alpha1/recognizer_types.go

```diff
@@ -0,0 +1,381 @@
+	// NOTYET: this field serves the same purpose as externalRef
+	// Name *string `json:"name,omitempty"`
+
+	// Output only. System-assigned unique identifier for the Recognizer.
+	// +kcc:proto:field=google.cloud.speech.v2.Recognizer.uid
+	UID *string `json:"uid,omitempty"`
+
+	// Default configuration to use for requests with this Recognizer.
+	//  This can be overwritten by inline configuration in the
+	//  [RecognizeRequest.config][google.cloud.speech.v2.RecognizeRequest.config]
+	//  field.
+	// +kcc:proto:field=google.cloud.speech.v2.Recognizer.default_recognition_config
+	DefaultRecognitionConfig *RecognitionConfigObservedState `json:"defaultRecognitionConfig,omitempty"`
+
+	// Output only. The Recognizer lifecycle state.
+	// +kcc:proto:field=google.cloud.speech.v2.Recognizer.state
+	State *string `json:"state,omitempty"`
+
+	// Output only. Creation time.
+	// +kcc:proto:field=google.cloud.speech.v2.Recognizer.create_time
+	CreateTime *string `json:"createTime,omitempty"`
+
+	// Output only. The most recent time this Recognizer was modified.
+	// +kcc:proto:field=google.cloud.speech.v2.Recognizer.update_time
+	UpdateTime *string `json:"updateTime,omitempty"`
+
+	// Output only. The time at which this Recognizer was requested for deletion.
+	// +kcc:proto:field=google.cloud.speech.v2.Recognizer.delete_time
+	DeleteTime *string `json:"deleteTime,omitempty"`
+
+	// Output only. The time at which this Recognizer will be purged.
+	// +kcc:proto:field=google.cloud.speech.v2.Recognizer.expire_time
+	ExpireTime *string `json:"expireTime,omitempty"`
+
+	// Output only. This checksum is computed by the server based on the value of
+	//  other fields. This may be sent on update, undelete, and delete requests to
+	//  ensure the client has an up-to-date value before proceeding.
+	// +kcc:proto:field=google.cloud.speech.v2.Recognizer.etag
+	Etag *string `json:"etag,omitempty"`
+
+	// Output only. Whether or not this Recognizer is in the process of being
+	//  updated.
+	// +kcc:proto:field=google.cloud.speech.v2.Recognizer.reconciling
+	Reconciling *bool `json:"reconciling,omitempty"`
+
+	// Output only. The [KMS key
+	//  name](https://cloud.google.com/kms/docs/resource-hierarchy#keys) with which
+	//  the Recognizer is encrypted. The expected format is
+	//  `projects/{project}/locations/{location}/keyRings/{key_ring}/cryptoKeys/{crypto_key}`.
+	// +kcc:proto:field=google.cloud.speech.v2.Recognizer.kms_key_name
+	KMSKeyName *string `json:"kmsKeyName,omitempty"`
+
+	// Output only. The [KMS key version
+	//  name](https://cloud.google.com/kms/docs/resource-hierarchy#key_versions)
+	//  with which the Recognizer is encrypted. The expected format is
+	//  `projects/{project}/locations/{location}/keyRings/{key_ring}/cryptoKeys/{crypto_key}/cryptoKeyVersions/{crypto_key_version}`.
+	// +kcc:proto:field=google.cloud.speech.v2.Recognizer.kms_key_version_name
+	KMSKeyVersionName *string `json:"kmsKeyVersionName,omitempty"`
```

Comment:
These two fields are awkward. But let's see how it goes when implementing the controller.